merlin/systems/dag/ops/fil.py

```diff
@@ -498,6 +468,14 @@ def save(self, version_path):
         model_path = pathlib.Path(version_path) / self.model_filename
         self.model.save_model(model_path)
 
+    def load(self, version_path) -> "LightGBM":
+        """Save model to version_path."""
+        model_path = pathlib.Path(version_path) / self.model_filename
+        if HAS_GPU:
+            self.model = cuml_fil.load(model_path, output_class=True)
```

Comment:
No, according to cuml it can load all cuml, xgboost, lightgbm and treelite, refer here: https://docs.rapids.ai/api/cuml/stable/api.html#cuml.ForestInference

Reply:
And treelite can also load lightgbm/xgboost models https://treelite.readthedocs.io/en/latest/treelite-api.html#treelite.Model.load